Core/Utilities/ObjectLoader.py

```diff
@@ -100,7 +100,7 @@ def __recurseImport(self, modName, parentModule=None, hideExceptions=False, full
       if impData[0]:
         impData[0].close()
     except ImportError as excp:
-      if str(excp).find("No module named %s" % modName[0]) == 0:
+      if "No module named" in str(excp) and modName[0] in str(excp):
```

Comment:
```suggestion
      if ("No module named %s" % modName[0]) in str(excp):
```
Otherwise it isn't completely the same test, if the exception message is  `"No module named Something"+modName[0] `
(Not sure if parenthesis needed or operator precedence is already enough

Reply:
This is intentional as the message is different.

Python 2:

```python
In [1]: import abcs
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-9ec4e562403a> in <module>()
----> 1 import abcs

ImportError: No module named abcs
```

Python 3:
```python
In [1]: import abcs
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-9ec4e562403a> in <module>
----> 1 import abcs

ModuleNotFoundError: No module named 'abcs'
```

Reply:
Wow, that is annoying.
```
# import re  # already done in the module
if re.findall("No module named '?%s'?" % modName[0], str(excp)):
```


```In [1]: import re

In [2]: s1 = "No module named abcs"

In [3]: s2=  "No module named 'abcs'"

In [4]: re.findall("No module named '?abcs'?", s1)
Out[4]: ['No module named abcs']

In [5]: re.findall("No module named '?abcs'?", s2)
Out[5]: ["No module named 'abcs'"]

In [6]: re.findall("No module named '?abcs'?", 'No module named abcsfoo')
Out[6]: ['No module named abcs'] # even the old test was broken in this regard, similar python3

In [7]: re.findall("No module named '?abcs'?", 'No module named fooabcs')
Out[7]: []
```